sdk/tests/conformance/context/context-attributes-alpha-depth-stencil-antialias.html

```diff
@@ -323,11 +323,14 @@
     gl.readPixels(0, 1, 1, 1, gl.RGBA, gl.UNSIGNED_BYTE, buf_2);
     pixel_1[0] = buf_1[0];
     pixel_2[0] = buf_2[0];
-    // For some anti-alias algorithms, effects may be not on diagonal line pixels, so that either:
-    //    - The red channel of the pixel at (0, 0) is not 0 and not 255, or,
-    //    - If it is 0, expect that the red channel of the pixel at (0, 1) is not 0 and not 255.
-    shouldBe("pixel_1[0] != 255 && pixel_1[0] != 0 || pixel_1[0] == 0 && pixel_2[0] != 255 && pixel_2[0] != 0",
-        "contextAttribs.antialias");
+    if (!contextAttribs.antialias) {
+        shouldBeTrue("pixel_1[0] == 0 && pixel_2[0] == 255");
+    } else {
+        // For some anti-alias algorithms, effects may be not on diagonal line pixels, so that either:
+        //    - The red channel of the pixel at (0, 0) is not 0 and not 255, or,
+        //    - If it is 0, expect that the red channel of the pixel at (0, 1) is 255.
+        shouldBeTrue("(pixel_1[0] != 255 && pixel_1[0] != 0) || (pixel_1[0] == 0 && pixel_2[0] == 255)");
```

Comment:
Exactly. I thought same concern when I was lying on the bed last night.
The new patch set resolves this issue.